testdata/ia/template.yaml

```diff
@@ -12,6 +12,10 @@ templates:
           type: float
           range: [0, 100]
           value: 80
+        - name: duration
+          summary: another test summary
+          unit: s
+          type: float
```

Comment:
Parameter without range and default value

Reply:
isn't it better to write this as summary?

Reply:
Better, fixed 👍